.ci/static-checks.sh

```diff
@@ -543,6 +543,7 @@ static_check_docs()
 		docs_status=$(echo "$docs_status" | grep "\.md$" || true)
 
 		docs=$(echo "$docs_status" | awk '{print $NF}' | sort)
+		docs=$(echo "${docs}" | grep -Ev "(${ignore_clh_generated_code})" | sort)
```

Comment:
`docs` is already sorted on the line above - do you need to sort it again after the `grep`?